includes/formFactory/ManageWikiFormFactoryBuilder.php

```diff
@@ -244,8 +248,11 @@ private static function buildDescriptorSettings(
 		$formDescriptor = [];
 
 		foreach ( $config->get( 'ManageWikiSettings' ) as $name => $set ) {
-			$add = ( $set['from'] == 'mediawiki' ) ||  in_array( $set['from'], $extList );
-			$disabled = ( $ceMW ) ? !( !$set['restricted'] || ( $set['restricted'] && $permissionManager->userHasRight( $context->getUser(), 'managewiki-restricted' ) ) ) : true;
+			$mwRequirements = isset( $set['requires'] ) ? ManageWikiRequirements::process( $set['requires'], $extList, false, $wiki ) : true;
```

Comment:
I'd prefer 'requires' to be a required param if we're making this part of the schema to match extensions.

Reply:
done